src/test/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetailsTest.java

```diff
@@ -151,6 +151,108 @@ public void testValidateResponseDetails_amongOptionsInvalidAnswer_shouldReturnNo
                 .validateResponsesDetails(Collections.singletonList(constantSumResponseDetails), 0).isEmpty());
     }
 
+    @Test
+    public void testShouldChangesRequireResponseDeletion_differentConstSumOptions_shouldReturnTrue() {
+        FeedbackConstantSumQuestionDetails feedbackQuestionDetails = new FeedbackConstantSumQuestionDetails();
+        feedbackQuestionDetails.setConstSumOptions(Arrays.asList("a", "b"));
+
+        FeedbackConstantSumQuestionDetails newFeedbackQuestionDetails = new FeedbackConstantSumQuestionDetails();
+        newFeedbackQuestionDetails.setConstSumOptions(Arrays.asList("a", "b", "c"));
+
+        assertTrue(feedbackQuestionDetails.shouldChangesRequireResponseDeletion(newFeedbackQuestionDetails));
+        assertTrue(newFeedbackQuestionDetails.shouldChangesRequireResponseDeletion(feedbackQuestionDetails));
```

Comment:
This is the only test case with two `assertTrue`s, I would suggest removing the extra `assertTrue` to make it consistent with the rest of the `testShouldChangesRequireResponseDeletion` tests.

Reply:
Thank you for the feedback! The idea behind of this test case was to test both conditions of the first if statement, one with the first list being bigger than the second and vice versa. If I remove one of these, the jacoco's report will show a yellow line meaning that just one of the conditions are being covered.

Reply:
> Thank you for the feedback! The idea behind of this test case was to test both conditions of the first if statement, one with the first list being bigger than the second and vice versa. If I remove one of those, the jacoco's report will show a yellow line meaning that just one of the conditions is being covered. Maybe I can split this test case in two others, each testing one of the conditions. What do you think about it?

Thanks for the clarification! There's no further changes needed in that case.